ocean_lib/data_provider/data_service_provider.py

```diff
@@ -360,13 +372,16 @@ def build_endpoint(service_name, provider_uri=None, config=None):
         parts = provider_uri.split('/')
         if parts[-2] == 'services':
             base_url = '/'.join(parts[:-2])
-            return f'{base_url}/services/initialize'
+            return "GET", f'{base_url}/services/initialize'
 
         api_version = DataServiceProvider.get_api_version()
         if api_version not in provider_uri:
             provider_uri = f'{provider_uri}/{api_version}'
 
-        return f'{provider_uri}/services/{service_name}'
+        service_endpoints = DataServiceProvider.get_service_endpoints()
+        method, url = service_endpoints[service_name]
+        url = url.replace('api/v1/', '')
+        return method, f'{provider_uri}{url}'
```

Comment:
please add an empty line before the return statement.
Will `api/v1/` always be the replacement? Could we possibly have another format? What will happen in that case?

Reply:
'api/v1/' occurs twice in url field.